docroot/sites/all/modules/custom/node_stat/node_stat.install

```diff
@@ -0,0 +1,38 @@
+<?php
+
+function node_stat_schema() {
+  $schema = array();
+  $schema['node_stat'] = array(
+    'description' => 'Information about node views.',
+    'fields' => array(
+      'nid' => array(
+        'type' => 'int',
+        'unsigned' => TRUE,
+        'not null' => TRUE,
+        'default' => 0,
+        'description' => "The node's {node}.nid.",
+      ),
+      'uid' => array(
+        'type' => 'int',
+        'unsigned' => TRUE,
+        'not null' => TRUE,
+        'default' => 0,
+        'description' => "The user's {uid}.uid.",
+      ),
+
+      'last_viewed' => array(
```

Comment:
I would call it timestamp, because it's not saving last viewed value for a node, it saves all views and the time they occurred